### PR TITLE
Update spec

### PR DESCRIPTION
**Pull request by ttoivone**
Many changes, but in this PR I didn't yet touch the execution steps for the operations. Also added a few Issues tagged with [ttoivone] that I think we need to address later.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -177,10 +177,11 @@ timestamp.
...
 
-A <dfn>VideoFrame</dfn> contains decoded video data. It will provide an
-{{ImageBitmap}} for rendering to {{Canvas}}. It should be possible to manipulate
-the {{ImageBitmap}} in {{WebGL}} and render the {{VideoFrame}} into a {{WebGL}}
-texture with texImage2D. It should also provide access to YUV data.
+A <dfn>VideoFrame</dfn> contains decoded video data. It can be drawn
+into {{Canvas}} with {{drawImage()}} or rendered into WebGL texture with
+{{texImage2D()}}.
+
+Issue: [ttoivone] shall access to raw YUV/RGB data be also allowed?
```
Comment (riju) [inline index.bs]: Better to link to this issue -
https://github.com/WICG/web-codecs/issues/45

File: index.bs [hunk 2]
```diff
@@ -267,8 +279,8 @@ The {{VideoFrame/codedWidth!!attribute}} and
 {{VideoFrame/codedHeight!!attribute}} constitute the production aperture of the
 image.
 
-Instances of {{VideoFrame}} are created with an internal slot <dfn attribute
-for=VideoFrame>\[[frame]]</dfn>.
+Issue: [ttoivone] need to specify in a section whether codedSize or visibleSize
```
Comment (riju) [inline index.bs]: Link to this issue https://github.com/WICG/web-codecs/issues/26

**Review by riju — APPROVED** [review]
Just nits :+1:

File: index.bs (hunk 1 above)
Comment (ttoivone) [reply index.bs]: Good point, I added the link. The issue #45 is actually reverse, creating the VideoFrame from YUV data while I was thinking about accessing a VideoFrame as an YUV data instead. But I'll update the issue.

File: index.bs (hunk 2 above)
Comment (ttoivone) [reply index.bs]: Done

**Review by riju — APPROVED** [review]

**Pull request merged**